dbt/adapters/spark_livy/livysession.py

```diff
@@ -86,6 +86,9 @@ def create_session(self, data):
         except requests.exceptions.JSONDecodeError as json_err:
             raise Exception("Json decode error to get session_id") from json_err
 
+        logger.debug(f"Obtained new livy session: {self.session_id}")
+        logger.debug("Polling to check if session is idle")
```

Comment:
nitpick. Add session id here also. so easy to debug when multithreaded.

Reply:
useful, added.

Reply:
updated sample log with the changes in description